src/iris/vendors/iris_hipchat.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+
+import logging
+import requests
+import time
+from iris.constants import IM_SUPPORT
+
+logger = logging.getLogger(__name__)
+
+
+class iris_hipchat(object):
+    supports = frozenset(['hipchat'])
```

Comment:
Can we add a `HIPCHAT_SUPPORT = 'hipchat'` to constants.py and use that here and in the iris.constants import above?